### PR TITLE
🎉 Adding Windows support

### PR DESCRIPTION
**Pull request by alexmercerind**
Finally I'll be adding support for PCs.
Checkout branch [windows](https://github.com/alexmercerind/harmonoid/tree/windows) for more information.

### CODE REVIEW

File: lib/globals.dart [hunk 1]
```diff
@@ -124,4 +124,5 @@ List<List<Color>> colors = [
   [Colors.green[700]             , Colors.green                  ],
 ];
 
+const String APP_DIR                                    = 'C:/Users/Alex Mercer/Music';
```
Comment (mytja) [inline lib/globals.dart:127]: You probably have to remove this here and change it to default user.
But I don't know anything about dart ;-)

**Review by mytja — COMMENTED** [review]
Sooooo......
This commit is some kind of mixture of C and Java (Dart)...
I don't know any of these languages
:laughing:

**Comment by alexmercerind** [comment]
😃 Hi @mytja,

This commit was just for testing & making mandatory changes to use the same code on Windows. 
**But, Flutter for Windows is still in pre-alpha development (they don't even have any guide for making modules). Right now I compiled the app for Windows (the .exe you know). But it looks like a mobile app scaled out on big screen. Window resizing is bad.**
Yeah, I need to change APP_DIR (I had to set it according to my username 😅 because there is not any path_provider for Windows yet.)

As you know that I'm using **audio_service** and **just_audio** for Android but...
There was not any audio player for Windows yet... So I was busy writing one myself (I have knowledge of C/C++ so it is a grea

**Review by mytja — APPROVED** [review]
Approve prerelease

File: lib/globals.dart (hunk 1 above)
Comment (alexmercerind) [reply lib/globals.dart:127]: Yeah 😅. There isn't a automatic way to do it on Windows yet. I'll try to write some more C++ or just ask user to set the directory.

Comment (alexmercerind) [reply lib/globals.dart:127]: Yes! There will be .exe files for Windows. The plan is to add support for Linux aswell.

**Comment by alexmercerind** [comment]
Flutter engine on Windows uses simple GLFW window (A simple game engine like), so you can expect great native performance. 
The app will not be slow, ram consuming and heavy like most electron apps out there. So that's a good thing.

**Comment by mytja** [comment]
Yeah, I like this app more on Android as based.
You should get some free time off this project.
When i release an LTS edition or major release on mine, i always go to other projects for at least a month.
And yeah, i saw your conversation with @raitonoberu about mp3
However i have so much work. All i wanted was a functional SpeechRecognition, and now i'm fixing a lot of other stuff (https://github.com/Uberi/speech_recognition/pull/513)

**Comment by alexmercerind** [comment]
Regarding this, I feel that I should use UWP as I'm kinda familiar with C# & XAML already.

It will be a great moment for me to experiment with native Windows 10 development. Since beginning, when I started programming I wanted to develop for Windows instead of Android or Flutter (but then other things started coming off). New fluent design for Windows is very beautiful & will give a decent app.

**Comment by alexmercerind** [comment]
@raitonoberu & @mytja !

After recalling all my C# memory & UWP experience. 😙

**Here is the first look of what Windows 10 can offer to the app in terms of look...**
harmonoid-pc-snippet


I know it is looking very "Generic" at the moment, but I have used my custom navigation menu in the left... So can customize it. Microsoft's new design language is fantastic.

Again, this is just a mid-development screenshot of what Harmonoid might look like on Windows 10 devices & performance is also very bad at the moment.
I can't make this public right now.

**Pull request closed without merging**